### PR TITLE
Ss-700

### PR DESCRIPTION
**Pull request by rsankagi**
Implementation of getting data one by one from SDK for admission usecase.

### CODE REVIEW

File: v1.0/frontend/SaralApp/android/saralsdk/src/main/java/org/ekstep/saral/saralsdk/SaralSDKOpenCVScannerActivity.java [hunk 1]
```diff
@@ -546,7 +623,7 @@ private void processCameraFrame(Mat image, long frameCount) {
 
             mStartPredictTime       = SystemClock.uptimeMillis();
             MediaActionSound sound  = new MediaActionSound();
-            sound.play(MediaActionSound.FOCUS_COMPLETE);
+            // sound.play(MediaActionSound.FOCUS_COMPLETE);
```
Comment (sanjay-v46) [inline v1.0/frontend/SaralApp/android/saralsdk/src/main/java/org/ekstep/saral/saralsdk/SaralSDKOpenCVScannerActivity.java:626]: why we need to comment this? pls revert

File: v1.0/frontend/SaralApp/android/saralsdk/src/main/java/org/ekstep/saral/saralsdk/SaralSDKOpenCVScannerActivity.java [hunk 2]
```diff
@@ -664,7 +770,7 @@ private void processScanningCompleted() {
         mScanningResultShared   = true;
 
         MediaActionSound sound  = new MediaActionSound();
-        sound.play(MediaActionSound.SHUTTER_CLICK);
+        // sound.play(MediaActionSound.SHUTTER_CLICK);
```
Comment (sanjay-v46) [inline v1.0/frontend/SaralApp/android/saralsdk/src/main/java/org/ekstep/saral/saralsdk/SaralSDKOpenCVScannerActivity.java:774]: revert this

Comment (rsankagi) [reply v1.0/frontend/SaralApp/android/saralsdk/src/main/java/org/ekstep/saral/saralsdk/SaralSDKOpenCVScannerActivity.java]: done

File: v1.0/frontend/SaralApp/android/saralsdk/src/main/java/org/ekstep/saral/saralsdk/SaralSDKOpenCVScannerActivity.java (hunk 1 above)
Comment (rsankagi) [reply v1.0/frontend/SaralApp/android/saralsdk/src/main/java/org/ekstep/saral/saralsdk/SaralSDKOpenCVScannerActivity.java]: done

**Review by sanjay-v46 — APPROVED** [review]

**Pull request merged**